guide/src/wasm-bindgen-test/continuous-integration.md

```diff
@@ -0,0 +1,73 @@
+# Setting Up Continuous Integration with `wasm-bindgen-test`
+
+This page contains example configurations for running `wasm-bindgen-test`-based
+tests in various CI services.
+
+Is your favorite CI service missing? [Send us a pull
+request!](https://github.com/rustwasm/wasm-bindgen)
+
+## Travis CI
+
+```yaml
+INSTALL_NODE_VIA_NVM: &INSTALL_NODE_VIA_NVM
+  |
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+    source ~/.nvm/nvm.sh
+    nvm install v10.5
+
+INSTALL_GECKODRIVER: &INSTALL_GECKODRIVER
+  |
+    curl --retry 5 -LO https://github.com/mozilla/geckodriver/releases/download/v0.21.0/geckodriver-v0.21.0-linux64.tar.gz
+    tar xf geckodriver-v0.21.0-linux64.tar.gz
+
+INSTALL_CHROMEDRIVER: &INSTALL_CHROMEDRIVER
```

Comment:
Instead of using this yaml syntax it may be best to inline it directly into the `install` section below to avoid confusing those who don't know about the syntax (which is probably almost everyone)